claudia-lab/app/scss/lib/_theme.scss

```diff
@@ -0,0 +1,8 @@
+$brand-black : #000;
+$brand-white : #fff;
+$brand-1st: #7afaf7;
```

Comment:
@loomnugget using numbers in your `_var` names is fine _but_ my preference is to avoid them - for style points